.github/workflows/python-tests.yml

```diff
@@ -2,6 +2,8 @@ name: Test
 
 on:
   pull_request:
+    branches:
```

Comment:
What is the point of doing it on pull request? Test should typically done on push, not pull.

Reply:
Currently I do not want to run unit tests in github workflow. It [downloads](https://github.com/cloudlinux/ansq/blob/cl/.github/workflows/python-tests.yml#L39) original nsqd (like other clients) without unix socket support and I just disable it from the any branches.